packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -396,6 +396,24 @@ export class Loan {
     }
     return await this.client.call('listauctions', [{ ...defaultPagination, ...pagination }], 'number')
   }
+
+  /**
+   * Returns information about auction history.
+   *
+   * @param {string} [owner] address or reserved word : mine / all (Default to mine)
+   * @param {ListAuctionHistoryPagination} pagination
+   * @param {number} [pagination.maxBlockHeight] Maximum block height
+   * @param {string} [pagination.vaultId] Vault Id
+   * @param {number} [pagination.index] Auction index
+   * @param {number} [pagination.limit = 100]
+   * @return {Promise<ListAuctionHistoryDetail>}
+   */
+  async listAuctionHistory (owner: string = 'mine', pagination?: ListAuctionHistoryPagination): Promise<ListAuctionHistoryDetail[]> {
```

Comment:
hmm, I know the vault id here is solely sorting index. But feel like list auction history as per vault is more useful for UI like `GET /auctions/:vaultId`